source/hic_hal/nuvoton/m48ssidae/StdDriver/src/m480_gpio.c

```diff
@@ -0,0 +1,146 @@
+/**************************************************************************//**
+ * @file     gpio.c
+ * @version  V3.00
+ * @brief    M480 series GPIO driver source file
+ *
+ * @copyright (C) 2011~2016 Nuvoton Technology Corp. All rights reserved.
```

Comment:
These files do not contain license header file, please add (entire StdDriver). Similar issue we had in Mbed OS. review all new files in this PR.

Reply:
Thanks for catching that. It’s fixed.